GovUk.Frontend.Umbraco.ExampleApp/Models/ModelsBuilder/GovukSummaryListItem.generated.cs

```diff
@@ -62,15 +62,15 @@ public GovukSummaryListItem(IPublishedElement content, IPublishedValueFallback p
 		///</summary>
 		[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Umbraco.ModelsBuilder.Embedded", "10.0.0+e3f4b86")]
 		[global::System.Diagnostics.CodeAnalysis.MaybeNull]
-		[ImplementPropertyType("itemKey")]
```

Comment:
This is a regression. This property was changed from 'key' to 'itemKey' to avoid a clash with the built-in Umbraco 'Key' property.